content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -143,27 +156,49 @@ is recommended to use
 [taints on nodes](/docs/concepts/configuration/taint-and-toleration/)
 to schedule those pods onto the right nodes.
 
-## PodSecurityPolicy Annotations
+## PodSecurityPolicy
+
+To control which sysctls can be set in pods, specify the
+`forbiddenSysctls` and/or `allowedUnsafeSysctls` fields in the PodSecurityPolicy.
+
+By default, all safe sysctls in the whitelist are allowed.
+
+Both `forbiddenSysctls` and `allowedUnsafeSysctls` are lists of plain sysctl names
+or sysctl patterns (which end with `*`). The string `*` matches all sysctls.
 
-The use of sysctl in pods can be controlled via annotation on the PodSecurityPolicy.
+The `forbiddenSysctls` field excludes specific sysctls, and can include a
+combination of safe and unsafe ones. To forbid setting any sysctls, use `*` on
+its own.
 
-Sysctl annotation represents a whitelist of allowed safe and unsafe sysctls
-in a pod spec. It's a comma-separated list of plain sysctl names or sysctl patterns
-(which end in `*`). The string `*` matches all sysctls.
+If you specify any unsafe sysctl in the `allowedUnsafeSysctls` field and it is
+not present in the `forbiddenSysctls` field, that sysctl can be used in Pods under
+this PodSecurityPolicy. In order to allow all unsafe sysctls in the PodSecurityPolicy
+to be set (except for those explicitly forbidden by `forbiddenSysctls`),
+use `*` on its own.
 
-Here is an example, it authorizes binding user creating pod with corresponding sysctls.
+It's not allowed to configure these two fields such that there is overlap.
```

Comment:
s/It's not allowed/Do not

What happens if you do? Is there an error?

Reply:
I think that my question is answered by the warning below. Maybe this sentence belongs as part of the warning.

Reply:
I still think the warning here is wrong. An overlap is simply not allowed by validation. Such a PSP is rejected. Nothing bad happens if you try.

Reply:
The warning and this sentence are unrelated. The warning is about pods which will be unable to launch even though the PSP allows them, but the kubelet does not. This sentence here is about the overlap of both fields in the PSP.